js/app.js

```diff
@@ -2,75 +2,80 @@ $(document).ready(function() {
   // all code to manipulate the DOM
   // goes inside this function
 // Declaring player1 and player 2
-var pl1 = [];
-var pl2 = [];
-var dogWin = 0;
-var catWin = 0;
-// Coding turns
-var currentTurn = 0;
-$('.box').one('click',function px(){
 
-if (checkWin()){
-	console.log();
-}
-    else if(currentTurn == 0){
+    // make sure to maintain your spacing and indentations! It makes your code easier
+    // for you to read and check for errors.
+    var pl1 = [];
+    var pl2 = [];
+    var dogWin = 0;
+    var catWin = 0;
+    // Coding turns
+    var currentTurn = 0;
 
-    $(this).append('<img id="theImg" img src="./1.png">');
-    pl1.push($(this).attr('id'));
-    checkWin();
-    currentTurn = 1;
-}
-else{
-    $(this).append('<img id="theImg" img src="./2.png">');
-    pl2.push($(this).attr('id'));
-    checkWin();
-    currentTurn = 0;
-}
+    // what is px()? Not a good name!
```

Comment:
Even tough it's a callback, the `px()` function name didn't make sense to me. Remember to make the code as readable as you can.